src/Commands/Install.php

```diff
@@ -52,14 +52,14 @@ class Install extends BaseCommand
     /**
      * The Command's Arguments
      *
-     * @var array
+     * @var array<string, string>
      */
     protected $arguments = [];
```

Comment:
I'm not sure you need `$arguments` at all, it probably comes from command `make:command`. However, the purpose of this PR was to fix the phpstan error problem.